libutils/known_dirs.c

```diff
@@ -23,11 +23,9 @@
 */
 
 #include <known_dirs.h>
-#include <cf3.defs.h>
+#include <definitions.h>
```

Comment:
Hmm, shouldn't this have been done earlier as part of the commit renaming that file?